era-compiler-solidity/.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[tools.clippy]
```

Comment:
Does it only work this way?
Maybe the root file is enough.

Reply:
It is, i just checked. Will move it to the root one